using-mobile-extensions/adobe-analytics/adobe-analytics-release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## June 1, 2020
+
+### @adobe/react-native-acpanalytics 1.1.6
```

Comment:
Are we including react versions here now?

Reply:
We started in Media so thought will do it here as well. Whatsay?

Reply:
I thought we removed the react version even for Media, didn't we?

Reply:
We just removed the react from the main release notes but kept in the media release notes.